pom.xml

```diff
@@ -49,7 +49,7 @@
     <dependency>
       <groupId>uk.gov.ons.ctp.product</groupId>
       <artifactId>collectionexercisesvc-api</artifactId>
-      <version>10.49.4</version>
+      <version>10.49.7-SNAPSHOT</version>
```

Comment:
The version should be `10.49.7`. I have fixed this in https://github.com/ONSdigital/rm-action-service/pull/21